DESCRIPTION

```diff
@@ -17,9 +17,19 @@ Depends:
     latticeExtra (>= 0.6-14),
     polynom,
     reshape
-Imports: car (>= 2.0-2), Hmisc, stats, utils, Rcpp (>= 0.12.17)
-Suggests: DEoptim, mgcv, chron, testthat, mco, cmaes, ppso, sensitivity,ggplot2,rsm,parallel,foreach,doRedis,ff,dream, knitr 
+Imports: 
+    car (>= 2.0-2),
+    Hmisc,
+    stats,
+    utils,
+    Rcpp (>= 0.12.17),
+    coda,
+    iterators
+Suggests: DEoptim, mgcv, chron, testthat, mco, cmaes, ppso, sensitivity,ggplot2,rsm,parallel,foreach,doRedis,ff,dream, knitr
```

Comment:
I noticed there was an importFrom parallel somewhere earlier, but it's listed as Suggests here. 
Does the importFrom only affect the NAMESPACE? R is a little confusing here.

Reply:
This shouldn't matter as `parallel` is part of r-core now, but it is a one of the integrated packages that needs to be loaded when you want to use it

Reply:
Yup, seems like `parallel` has been a part of R base since v 3.6.2, so it'll be available to the user regardless of whether or not its listed as Suggests.

Adding the `@importFrom package function` tag in a .R file will make sure that particular function can actually be used by your code, and corresponding importFrom() tag will appear in the `NAMESPACE` after applying `roxygen2::roxygenise`.